### ISSUE TITLE
lk_target_loss return None, scale of batch_locs? How to set forward_max and fb_thresh?

### ISSUE THREAD

**Issue by zhaoruiqiff** [D-X-Y/landmark-detection#73]
Thank you for great work!

I'm trying to fine tune my pretrained model using SBR. I have a question about the data scale. 

For **images**, should we **normalize** the pixel values to 0-1 and further normalize using the following parameters for each channel?
**mean**=[0.485, 0.456, 0.406],
**std**=[0.229, 0.224, 0.225]

For predicted points locations (batch_locs, batch_next, batch_fbak, batch_back) in order to compute **lk_target_loss**, are **batch_locs** pixel locations in the image coordinate? 
For example if my image size is 256*256, batch_locs will be from 0-255?

Since I'm training on another video dataset, how should I pick **forward_max** and **fb_thresh**? 
In your example  **fb_thresh = 1**, **forward_max = 2**, they seem too small compared to my image scale of 256*256, so **sequence_checks** all become False after the loops and all the losses are masked out, so I can't get any loss, but None. Can you guide me here what I did wrong? 
What should the scale of predicted points locations **batch_locs** in order to compute **lk_target_loss**? 
How should I choose **forward_max** and **fb_thresh**?

Thanks!

**Comment by zhaoruiqiff** [comment]
Thanks for answering my questions! I have been able to make the code work.

I tried the SRT code, and used the SBR loss as defined in temporal_loss_regression.py. I used my pretrained model as initialization, and fine tuned on my video dataset for a few epochs, using both the SBR loss and my regression loss. I used the same setting as in REG-300W-SBR-300VW-P68.sh

Currently based on my metric, the landmark detection stability on video improved by about 16%. Before fine tuning using your SBR loss, I think visually my results already looked satisfying. So this is quite an improvement. Thanks for your great work! The demo video on the github page looks very impressive. I want to know how to make my detection results on video look as impressive as that one? How should I train my model on videos? Any important parameters to adjust?

Thank you!
